tests/unit/commandlog.tcl

```diff
@@ -120,6 +119,9 @@ start_server {tags {"commandlog"} overrides {commandlog-execution-slower-than 10
         assert_equal {foobar} [lindex $e 5]
 
         # for large-reply
+        set copy_avoid [lindex [r config get min-io-threads-avoid-copy-reply] 1]
+        r config set min-io-threads-avoid-copy-reply 0
```

Comment:
This is the only one that I'm still not clear about. Do we not report large replies that are offloaded? We probably should.

Reply:
Ok, do we want to do this before the merge, or follow up?